pulser/register.py

```diff
@@ -150,9 +150,7 @@ def _draw_2D(
         if with_labels:
             # Determine which labels would overlap and merge those
             plot_pos = list(pos[:, (ix, iy)])
-            plot_ids = [
-                f"[{i}]" if i in masked_qubits else f"{i}" for i in ids
-            ]
+            plot_ids: list[Union[list, str]] = [[f"{i}"] for i in ids]
```

Comment:
The type here is now `list[list[str]]`, right?

Reply:
Yes, but later on the sublists become strings. If I don't force the `list[Union[list, str]]` type then mypy complains

Reply:
Ok, roger that